### DISCUSSION TITLE
Upcoming Webinar 11/30/2021

### DISCUSSION THREAD

**Discussion by shaselton-usds** [CMSgov/price-transparency-guide#279] [Announcements]
Please join the Centers for Medicare & Medicaid Services (CMS) on Tuesday, November 30, 2021 for the third in a series of technical implementation webinars for the in-network negotiated rate and out-of-network historical billed charges machine-readable file requirements in the Transparency in Coverage (TiC) Final Rules (85 FR 72158).

During the third webinar, the CMS team will recap November's webinar, cover new decisions arrived on GitHub and provide insight on file size. CMS will also respond to appropriate questions that are posted in this announcement as well as in the Q&A chat box during the webinar.

Please register for the webinar [here](https://cms.zoomgov.com/webinar/register/WN_vDbjlj5nQ7y1ddQUdROzsg).

Post questions below and like/upvote questions so the team knows what's most important to you.

**Comment by MMath19103** [comment]
Please clarify the expectations for OON Allowed Amount 20 Claim Threshold Determination; I've posted a question #275 on the discussion board. The previous question #181 in my view was not clear. We are trying to finalize our processing rules and this remains an open question. Thank you in advance.

**Reply by kavkuk** [reply]
@shaselton-usds agree with @MMath19103 . Hoping this will be discussed tomorrow, as we also need to finalize our logic and rules around this requirement.

**Comment by Devyandu** [comment]
Guidance on two topics please.
- Use of reference files.
- Guidance on file naming convention.

PS : Even though final details may not be available yet, it would be good to know if we are going in the direction on reference files. That would mean payers can stop development work on ingesting and staging partner network data.

**Comment by SODay-BA** [comment]
The specification for the in-network object indicates that Billing Code Type Version is required.  However, there is no version number for some common billing code types (HCPCS, CPT), and the specification even acknowledges that a billing code type "might" have a version number.  Is it acceptable to just utilize the 4-digit year of the date of service as a substitute for an actual version number?      
From https://github.com/CMSgov/price-transparency-guide/blob/master/schemas/in-network-rates/README.md                
Field:  billing_code_type_version
Name:  Billing Code Type Version
Type:  String
Definition:  There might be versions associated with the billing_code_type. For example, Medicare's current (as of 5/24/21) MS-DRG version is 37.2
Required:  Yes

**Comment by cpstark** [comment]
What is the intention of reporting rates for procedures which are heavily based on modifiers, such as imaging? With no field for a modifier, are we to report the global rate only? (example- CPT 71010 modifier 26 is the professional read only for chest x-ray.) See open discussion #12.

**Comment by kavkuk** [comment]
In scenarios where we have only the provider SSN (either provider is in the process of getting an NPI or is just using his SSN for TIN and does not have NPI), what are we to provide in the provider_groups object required field since we dont want to expose the SSN?  Solution for  #73 does not apply as provider does not have NPI.

**Comment by taylorpatriciab** [comment]
@shaselton-usds is the ppt you displayed on the call available?

**Reply by shaselton-usds** [reply]
It is. I can be downloaded here: https://www.cms.gov/healthplan-price-transparency/resources/webinars 

Let me know if you run into any issues.

**Reply by taylorpatriciab** [reply]
@shaselton-usds I was able to download.  thanks!

**Comment by CTurner72** [comment]
Will Q & A shared via chat during the 11/30 call be added here too?  Thank you.

**Comment by tdfow** [comment]
@shaselton-usds is there a target on when the pull requests discussed in the webinar will either be incorporated into the schema or is there potential other changes for those two pull requests discussed?
